liquibase-core/src/main/java/liquibase/change/core/AbstractModifyDataChange.java

```diff
@@ -59,7 +60,7 @@ public void setWhere(String where) {
     /**
      * @deprecated use getWhere().
      */
-    @DatabaseChangeProperty(isChangeProperty = false)
+    @DatabaseChangeProperty(isChangeProperty = false, description = "Deprecated. Use 'where'")
```

Comment:
For this kind of deprecated method for which we already have a newer version. Don't we need to use `@Deprecated` annotation? @nvoxland (cc: @filipelautert )